troposphere/static/js/components/admin/ImageRequest.react.js

```diff
@@ -136,141 +136,143 @@ export default React.createClass({
 
         return (
         <div className="request-admin pull-right admin-detail">
-            <div>
-                <h3>Request Information</h3>
+            <div style={{ marginBottom: "20px" }}>
+                <h3 className="t-body-2">Request Information</h3>
                 <div>
-                    Request ID:
-                    {request.get("id")}
+                   Request ID:
+                   {` ${request.get("id")}` }
```

Comment:
True,
There were a lot of lines on this file to change, that was just the quickest way to get the missing space using `.` in vim. I can change it.